coralproject/talk
# One build

## What does this PR do?

- Now you can build all the files from the top level of the project.
- Routing works better. You can hit `http://localhost:3000/client/coral-admin` and the links work from there, but if you go to `/configure` first, or reload, the server doesn't know what to do. We have to set up some routing in the express app.

### Caveats
- styles still do not work yet. I'm branching off here to work on that as we speak.

## How do I test this PR?

- `npm run build` from the root. both `coral-embed-stream` and `coral-admin` should build.
- go to `http://localhost:3000/client/coral-admin`. The menu links on top should work.

@coralproject/tech
